docs/server/source/simple-network-setup.md

```diff
@@ -252,6 +252,66 @@ If you want to refresh your node back to a fresh empty state, then your best bet
 - reset Tendermint using `tendermint unsafe_reset_all`
 - delete the directory `$HOME/.tendermint`
 
+## Shutting down BigchainDB
```

Comment:
This section is way too detailed for a "simple network setup". You can move some of the details in other parts of the documentation.

Reply:
Maybe just create a new stand-alone page about how to shut down BigchainDB, and then link to that page from here?

Reply:
@ttmc @vrde should I stick to creating a new page and referring it here?